src/Steps.cpp

```diff
@@ -1001,6 +1001,14 @@ RString Steps::GenerateChartKey(NoteData &nd, TimingData *td)
 	}
 	k = firstHalf + secondHalf;
 
+	Group* pGroup = SONGMAN->GetGroupByName(m_pSong->m_sGroupName);
```

Comment:
Where are GenerateChartKey() and GetChartKey() being used? genuine question as I didn't see anywhere where these functions were being called so im curious what this changes